cmd/db_list.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/anchore/grype/grype/db"
+	"github.com/spf13/cobra"
+)
+
+var dbListOutputFormat string
+
+var dbListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "list all DBs available according to the listing URL",
+	Args:  cobra.ExactArgs(0),
+	RunE:  runDBListCmd,
+}
+
+func init() {
+	dbListCmd.Flags().StringVarP(&dbListOutputFormat, "output", "o", "text", "format to display results (available=[text, raw, json])")
+
+	dbCmd.AddCommand(dbListCmd)
+}
+
+func runDBListCmd(_ *cobra.Command, _ []string) error {
+	dbCurator, err := db.NewCurator(appConfig.DB.ToCuratorConfig())
+	if err != nil {
+		return err
+	}
+
+	listing, err := dbCurator.ListingFromURL()
+	if err != nil {
+		return err
+	}
+
+	supportedSchema := dbCurator.SupportedSchema()
+	available, exists := listing.Available[supportedSchema]
+
+	if len(available) == 0 || !exists {
+		fmt.Printf("No databases available for the current schema (%d)\n", supportedSchema)
```

Comment:
nit: this may be subjective, but this seems like the kind of message that should be routed to stderr. What do you think?

Reply:
I'm a bit torn --yes, in the sense that does align with my opinion. But I wouldn't update it just for this command, I would also update all other commands (only the root command has this preference too, all of the others print to stdout  only).

Reply:
added a helper and started using that in all aux commands